frontend/src/components/search/components/common/video-tag/video-tag.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC } from 'common/types/types';
+import styles from './styles.module.scss';
+
+type Props = {
+  name: string;
+};
+
+const VideoTag: FC<Props> = ({ name }) => (
+  <>
+    <div className={styles['video-tag']}>{name}</div>
+  </>
```

Comment:
You do not need react fragment here.

Better to have styles container and an additional span for text